source/documentors/references/doc_style_guide.rst

```diff
@@ -0,0 +1,412 @@
+- Endanger the user's data or their solution.
+- Lead to inappropriate, undesired, or unexpected results within the user's solution.
+- Expose the user or their solution to risk.
+
+Example:
+
+.. warning:: Vital information that should not be ignored. The user incurs risk if this information is not followed. Please, check this section of the RST guide to know how to include them.
+
+See also
+========
+
+This directive is useful for referencing other documents related to the topic that may be of interest to the reader.
+
+.. seealso:: Here goes the reference to another document.
+
+Additionally, “See Also Tables” is an important way for users to find documents related to the topic they are exploring. Good docs will have thorough, accurate, and relevant links in the See Also section through this syntax.
+
+.. seealso::
+    :class: dropdown
+
+    :ref:`Offering Differentiated Content` (concept)
+    :ref:`Configure Your Course for Content Experiments` (how-to)
+
+Table of contents
+=================
+
+The table of contents is the navigation section to the left of your document. Clicking entries on the table of contents will open that document. The table of contents is consistent across all documents on docs.openedx.org. Every document must belong to at least one table of contents. In other words, you cannot create “orphan” documents that are not reachable to users via standard navigation.
+
+As an example, if a new file `build_a_course.rst` is created, then it would need to be appended to an existing table of contents, as shown in the example below. 
+
+You can learn more about adding your document to a table of contents by following this guide.
+
+Also, you can do a :glob: `*` on an index page, meaning in some cases, documents in a page tree are automatically added to the ToC. In the following example, any file contained in the same folder will automatically be added to the table of contents. It is a quick way to set up a table of contents so that new documents are automatically picked up, but it reduces the control you have on the order of your documents in the TOC:
+
+.. code-block::
+
+    .. toctree::
+       :maxdepth: 1
+       :glob:
+
+       *
+
+Tables
+======
+
+When you include a table, be sure to include a heading row. In addition, consider whether a stub column is appropriate. The heading row and stub column provide useful context for users of screen readers.
+
+Tables should be used to compile complicated data and indicate its relationships. In Open edX documentation, tables look like this:
+
+.. list-table:: Title
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Heading row 1, column 1
+     - Heading row 1, column 2
+     - Heading row 1, column 3
+   * - Row 1, column 1
+     -
+     - Row 1, column 3
+   * - Row 2, column 1
+     - Row 2, column 2
+     - Row 2, column 3
+
```

Comment:
I think you're missing an intro to the bulleted list, maybe, "When to use a table"?

Reply:
Yes! I've added!